add_gem_package.sh

```diff
@@ -78,14 +78,9 @@ add_to_tito_props() {
 
 add_gem_to_all_comps() {
 	add_gem_to_comps $TITO_TAG
-	if [[ $TITO_TAG == "foreman-nightly-rhel7" ]] || [[ $TITO_TAG == "foreman-nightly-nonscl-rhel7" ]] ; then
-		add_gem_to_comps foreman-nightly-el8
-	elif [[ $TITO_TAG == "foreman-plugins-nightly-rhel7" ]] || [[ $TITO_TAG == "foreman-plugins-nightly-nonscl-rhel7" ]] ; then
-		add_gem_to_comps foreman-plugins-nightly-el8
-	elif [[ $TITO_TAG == "foreman-client-nightly-rhel7" ]] ; then
+	if [[ $TITO_TAG == "foreman-client-nightly-rhel7" ]] ; then
```

Comment:
Thinking out loud: should we recommend always passing EL8 as a base and make the scripts work with that?